### PR TITLE
Update destination tester README

### PR DESCRIPTION
**Pull request by fivetran-abdulsalam**
Links: https://fivetran.height.app/T-738840
Add CLI args detail in README

### CODE REVIEW

File: tools/destination-tester/README.md [hunk 1]
```diff
@@ -153,3 +153,9 @@ This option disables encryption and compression of batch files for debugging pur
 
 #### --input-file
 The tester by default reads all input files from local data folder and executes them in the alphabetical order they appear. You can specify a single input file to be read and executed using this option. Providing just the filename is sufficient.
+
+#### --schema-name
+The tester by default creates `tester` named schema. This option tells the tester to create a different schema name than the default.
+
+#### --disable-operation-delay
+The tester by default delays operations to simulate a real sync. This boolean option tells the tester to not simulate a real sync.
```
Comment (manjutapali) [inline tools/destination-tester/README.md:161]: The tester by default adds delay to operation for real time simulation,  set this boolean to disable delay.

**Review by manjutapali — APPROVED** [review]
Looks good.

File: tools/destination-tester/README.md [hunk 2]
```diff
@@ -153,3 +153,9 @@ This option disables encryption and compression of batch files for debugging pur
 
 #### --input-file
 The tester by default reads all input files from local data folder and executes them in the alphabetical order they appear. You can specify a single input file to be read and executed using this option. Providing just the filename is sufficient.
+
+#### --schema-name
+The tester by default creates `tester` named schema. This option allows the tester to run with different schema name.
```
Comment (5tran-alexil) [inline tools/destination-tester/README.md:158]: @fivetran-abdulsalam Is the schema named `tester`, literally, or `tester` is used as a prefix or suffix, or the actual name is passed as a tester argument?

Comment (fivetran-abdulsalam) [reply tools/destination-tester/README.md:158]: It is literally `tester`

Comment (5tran-alexil) [inline tools/destination-tester/README.md:158]:
```suggestion
The tester by default creates a schema named `tester`. This option allows the tester to run with a custom schema name by specifying `--schema-name <custom_schema_name>` where `<custom_schema_name>` is your custom schema name. 
```

File: tools/destination-tester/README.md [hunk 3]
```diff
@@ -153,3 +153,9 @@ This option disables encryption and compression of batch files for debugging pur
...
+
+#### --disable-operation-delay
+The tester by default adds delay to operation for real time simulation, set this boolean to disable delay.
```
Comment (5tran-alexil) [inline tools/destination-tester/README.md:161]:
```suggestion
The tester by default adds a delay to operation for real-time simulation. Specifying this argument disables the delay.
```

**Review by 5tran-alexil — COMMENTED** [review]
See my suggestion.

File: tools/destination-tester/README.md (hunk 3 above)
Comment (5tran-alexil) [inline tools/destination-tester/README.md:161]: @fivetran-abdulsalam 
'--disable-operation-delay' - what is the syntax for setting this key to 'true'?
Or just specifying this key disables the delay?

Comment (fivetran-abdulsalam) [reply tools/destination-tester/README.md:161]: [@alexander.ilyichov](https://fivetran.slack.com/team/U01AUGQFYHK)
 just specifying this key will disable it.
e.g.
docker run --mount type=bind,source=<local-data-folder>,target=/data -a STDIN -a STDOUT -a STDERR -it -e WORKING_DIR=<local-data-folder> -e GRPC_HOSTNAME=host.docker.internal --network=host fivetrandocker/sdk-destination-tester:<version> --disable-operation-delay

**Review by 5tran-alexil — APPROVED** [review]
LGTM

**Pull request merged**